apps/codeforafrica/src/pages/projects/[slug].page.js

```diff
@@ -16,9 +17,18 @@ import TeamMemberCardList from "@/codeforafrica/components/TeamMemberCardList";
 import { projects, getPageStaticProps } from "@/codeforafrica/lib";
 
 function Index({ project, sections, ...props }) {
+  const { badges } = project;
   return (
     <Page {...props}>
       <ProjectPageHeader {...project} />
+      <Section
+        sx={{
+          marginTop: { xs: "26.6px", sm: "20px", md: "56px" },
+          marginBottom: "42px",
+        }}
```

Comment:
Something is not right with these... the accolade badge list doesn't align well with the rest of components on the page.

**Mobile**
![Screenshot from 2022-06-10 12-31-00](https://user-images.githubusercontent.com/1779590/173036649-91cf5c9f-29f0-46d5-b80e-2f26aaa43860.png)

**Tablet**
![Screenshot from 2022-06-10 12-31-38](https://user-images.githubusercontent.com/1779590/173036669-6baa16b9-be76-4092-8c48-bd6f571da434.png)

On desktop too.